src/stateToParams.js

```diff
@@ -24,7 +24,7 @@ export function stateToParams(initialState, currentState, location) {
       isDefault = typeof(currentItemState) === 'object' ? isEqual(initialValue, currentItemState) : currentItemState === initialValue;
     }
     // if it is default or doesn't exist don't make a query parameter
-    if (((!currentItemState && !options.serialize) || isDefault) && !options.setAsEmptyItem) {
+    if (((typeof(currentItemState) === undefined && !options.serialize) || isDefault) && !options.setAsEmptyItem) {
```

Comment:
I tried this in the console and I think the syntax is wrong here. I think it should be `typeof currentItemState === "undefined" && ...`
<img width="282" alt="Screen Shot 2020-01-21 at 1 56 22 PM" src="https://user-images.githubusercontent.com/5234453/72846679-eb8d1580-3c55-11ea-94f9-fdccbea0944e.png">

Reply:
Oh, yes. Of course. "undefined"

Reply:
with brackets -> 
![image](https://user-images.githubusercontent.com/27211707/72884792-c7920880-3d17-11ea-833d-59455ab56366.png)

:)